src/index.js

```diff
@@ -145,7 +145,7 @@ export default function fetch(url, opts) {
 				statusText: res.statusMessage,
 				headers: headers,
 				size: request.size,
-				timeout: request.timeout
+				timeout: opts && opts.bodyTimeout || request.timeout
```

Comment:
what differences dose this make?
The only thing you have done is making an alias for `opts.timeout` and added a test that probably already exist somewhere. 

Also I would discourage the use of such options and instead use the AbortController when it lands since timeout isn't part of the spec

For Example

# using abortController to control timeout for initial response header

```js
const controller = new AbortController()
const { signal } = controller
const timer = setTimeout(controller.abort, 5000)

fetch(url, { signal })
  .then(res => {
    // everything uploaded, starting to download something

    // Stop the timer from aborting something we are downloading
    clearTimeout(timer)
  })
```

# Or using abort controller with timeout for the hole upload and download

```js
const controller = new AbortController()
const { signal } = controller
const timer = setTimeout(controller.abort, 5000)

fetch(url, { signal })
  .then(res => {
    // everything uploaded, starting to download something
    // now waiting for the final response (leaving the timer be)
    return res.text()
  }).then(res => {
    // downloaded everything in time
  })
```


# Or starting a timeout if something you are downloading is very slow

```js
const controller = new AbortController()
const { signal } = controller

fetch(url, { signal })
  .then(res => {
    // everything uploaded, starting to download something
    // abort if it's going to slow
    const timer = setTimeout(controller.abort, 5000)
    // can use content-length to figure out roughly how long it shold be
    return res.text()
  }).then(res => {
    // downloaded everything in time
  })
```